nectar/downloaders/threaded.py

```diff
@@ -20,7 +21,7 @@
 DEFAULT_MAX_CONCURRENT = 5
 DEFAULT_BUFFER_SIZE = 8192  # bytes
 DEFAULT_PROGRESS_INTERVAL = 5  # seconds
-DEFAULT_TRIES = 2
+DEFAULT_TRIES = 5
 
```

Comment:
I picked a few values out of thin air, honestly. 

I did think it was worth upping the default tries since we blacklist the hostname after enough failures, and since downloaders are pretty much exclusively used with a single hostname that really means the downloader is permanently done. This, of course, makes one wonder why we would keep the blacklist feature at all, but that's a different issue.